apps/common-app/src/examples/RuntimeTests/tests/animations/withDelay/addDelays.test.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect } from 'react';
+import { View, StyleSheet } from 'react-native';
+import Animated, { useSharedValue, useAnimatedStyle, withTiming, withDelay } from 'react-native-reanimated';
+import {
+  describe,
+  test,
+  render,
+  useTestRef,
+  getTestComponent,
+  wait,
+  mockAnimationTimer,
+  recordAnimationUpdates,
+  unmockAnimationTimer,
+  expect,
+  clearRenderOutput,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+const WIDTH_COMPONENT_ACTIVE_REF = 'WidthComponentActive';
+const WIDTH_COMPONENT_PASSIVE_REF = 'WidthComponentPassive';
+const WIDTH_COMPONENT_INLINE_REF = 'WidthComponentInline';
+const COMPONENT_REFS = [WIDTH_COMPONENT_ACTIVE_REF, WIDTH_COMPONENT_PASSIVE_REF, WIDTH_COMPONENT_INLINE_REF];
+
+describe('Add three delays', () => {
+  const WidthComponent = ({ delays }: { delays: [number] | [number, number, number] }) => {
```

Comment:
Can you move component definition outside `description` invocation? You did it for `keepSnapshot.test`